bioimageio/spec/shared/fields.py

```diff
@@ -225,7 +225,15 @@ class Raw(DocumentedField, marshmallow_fields.Raw):
 
 
 class String(DocumentedField, marshmallow_fields.String):
-    pass
+    def __init__(self, *, min_length: int = 1, **super_kwargs):
+        validate = super_kwargs.get("validate", None)
+        if isinstance(validate, typing.Iterable):
```

Comment:
I suppose the `validate is None` case is missing...
```suggestion
        if validate is None:
            validate = []
        elif isinstance(validate, typing.Iterable):
```